arc42/05-Bausteinsicht.adoc

```diff
@@ -1 +1,28 @@
 = Bausteinsicht
+
+== Grobe Übersicht
+
+Unsere Anwendung besteht aus drei elementaren Schichten: +
+Model , Services und infrastructure. +
+
+* In der infrastructure-Schicht befinden sich im wesentlichen Controller, die Anfragen über den Webbrowser entgegennehmen. +
+
+* In der Services-Schicht befinden sich verschidene Services, die in den Controller benutzt werden und verschiedene Dienste leisten. +
+
+* In der Model-Schicht befinden sich unsere Anwendungsobjekte,die die grundlegende Komponente unseres Projektes realisieren. +
+
+== Post-Anfragen
+
+Im Falle von Post-Anfragen bzw. Formularen wandelt der Controller die Formulardaten in Objekte des Modells um.
+Diese Objekte werden falls nötig von den Services weiter verarbeitet und schließlich von dem entsprechenden Repository in
+die Datenbank gespeichert.
+
+== GET-Anfragen
```

Comment:
Please ensure a uniformly design. When you don't use capital letters anywhere, please don't do it here too.
```suggestion
== Get-Anfragen
```